tests/unit_tests/model/test_telescope_model.py

```diff
@@ -7,26 +7,16 @@
 import pytest
 from astropy import units as u
 
-import simtools.utils.general as gen
 from simtools.model.telescope_model import InvalidParameter, TelescopeModel
 
 logger = logging.getLogger()
 logger.setLevel(logging.DEBUG)
 
 
 @pytest.fixture
```

Comment:
Is this really needed just to return a string of the filepath?

Reply:
pytest fixtures are very useful in general (I won't give details here, but it's worth reading about). In this case it is used as a nice way to define a constant for all tests in the module that need it. It is not really the intended use case and It is much more powerful than that, but since we use it everywhere for other things, we might as well use it here as well.